cypress/integration/pages/testsForAllPages.js

```diff
@@ -230,15 +230,34 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
     });
 
     describe('Header Tests', () => {
+      const localizedName =
```

Comment:
It'd be good to rename this, to make it clear it's a boolean
```suggestion
      const hasLocalisedName =
```